easybuild/easyconfigs/m/mpiP/mpiP-3.4.1-iimpi-2019a.eb

```diff
@@ -0,0 +1,49 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+easyblock = 'ConfigureMake'
+
+name = 'mpiP'
+version = '3.4.1'
+
+homepage = 'http://mpip.sourceforge.net/'  # https:// doesn't work
+description = """
+ mpiP is a lightweight profiling library for MPI applications.  Because it
+ only collects statistical information about MPI functions, mpiP generates
+ considerably less overhead and much less data than tracing tools.  All the
+ information captured by mpiP is task-local. It only uses communication during
+ report generation, typically at the end of the experiment, to merge results
+ from all of the tasks into one output file.
+"""
+
+toolchain = {'name': 'iimpi', 'version': '2019a'}
+toolchainopts = {'usempi': True, 'pic': True, 'strict': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['688bf37d73211e6a915f9fc59c358282a266d166c0a10af07a38a01a473296f0']
+
+builddependencies = [
+    ('binutils', '2.31.1', '', ('GCCcore', '8.2.0')),
+]
+dependencies = [
+    ('libunwind', '1.3.1', '', ('GCCcore', '8.2.0')),
+    ('zlib', '1.2.11', '', ('GCCcore', '8.2.0')),
+]
+
+preconfigopts = 'CPPFLAGS="$CPPFLAGS -DPACKAGE -DPACKAGE_VERSION"'
+
+buildopts = " && make shared"
+
+installopts = 'install-all'
+
+modextrapaths = {'LD_PRELOAD': ['lib/libmpiP.so']}
```

Comment:
@JackPerdue Please use `SHLIB_EXT` constant rather than hardcoding `.so` (like you do in `mpiP-3.4.1-gompi-2019a.eb `)